packages/flutter/test/material/buttons_test.dart

```diff
@@ -1191,4 +1192,45 @@ void main() {
     final Material material = tester.widget<Material>(rawButtonMaterial);
     expect(material.color, const Color(0xff00ff00));
   });
+
+  group('feedback', () {
```

Comment:
Could you add a set of these tests for `RawMaterialButton` as well?

Reply:
Done